UnityProject/Assets/Scripts/Player/GhostMove.cs

```diff
@@ -13,11 +13,11 @@ public class GhostMove : NetworkBehaviour, IPlayerControllable
 
 	public bool Moving = false;
 
-	private RegisterTile registerTile;
+	public RegisterTile registerTile;
```

Comment:
```suggestion
	public RegisterTile RegisterTile {get; private set;};
```

Can you do that for all of these

Reply:
I was  Settingit through the editor

Reply:
```suggestion
	public RegisterTile RegisterTile {get; set;};
```
Do that then

Reply:
oh ok, i cant read

Reply:
Cant you just [SerialiseField] then?

Reply:
ok